app/main.py

```
@@ -85,6 +88,28 @@ def ping():
+    DB 등 핵심 의존성의 상태를 점검합니다. 
+    장애 시 503 상태 코드를 반환하여 로드밸런서가 비정상 상태를 식별할 수 있도록 합니다.
+    """
+    health_status = {"status": "healthy", "dependencies": {"database": "ok"}}
+    try:
+        supabase = get_supabase_client()
+        # 2초 타임아웃 제한 (동기 I/O를 worker thread로 위임하여 이벤트 루프 블로킹 방지)
+        await asyncio.wait_for(
+            asyncio.to_thread(supabase.table("room").select("id").limit(1).execute),
+            timeout=2.0
+        )
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)